app/validators/national_health_index_validator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'nhi_validator'
+
+class NationalHealthIndexValidator < ActiveModel::Validator
+  def validate(record)
+    record.errors[:nhi] << (options[:message] || 'is not a valid NHI') unless NHIValidator.valid?(record.nhi)
+  end
+
+  def validate_each(record, attribute, value)
+    record.errors[attribute] << (options[:message] || 'is not a valid NHI') unless NHIValidator.valid?(value)
```

Comment:
no tests for this line either

Reply:
Right - we don't actually need the `validate_each` section, I'll remove it